hedera-node/hedera-config/src/main/java/com/hedera/node/config/data/BootstrapConfig.java

```diff
@@ -16,28 +16,36 @@
 import com.hedera.pbj.runtime.io.buffer.Bytes;
 import com.swirlds.config.api.ConfigData;
 import com.swirlds.config.api.ConfigProperty;
 
 @ConfigData("bootstrap")
 public record BootstrapConfig(
-        @ConfigProperty(value = "feeSchedulesJson.resource", defaultValue = "feeSchedules.json")
+        @ConfigProperty(value = "feeSchedulesJson.resource", defaultValue = "feeSchedules.json") @NetworkProperty
                 String feeSchedulesJsonResource,
         @ConfigProperty(
                         value = "genesisPublicKey",
                         defaultValue = "0aa8e21064c61eab86e2a9c164565b4e7a9a4146106e0a6cd03a8c395a110e92")
+                @NetworkProperty
                 Bytes genesisPublicKey,
         @ConfigProperty(value = "hapiPermissions.path", defaultValue = "data/config/api-permission.properties")
+                @NetworkProperty
                 String hapiPermissionsPath,
         @ConfigProperty(value = "networkProperties.path", defaultValue = "data/config/application.properties")
+                @NetworkProperty
                 String networkPropertiesPath,
-        @ConfigProperty(value = "rates.currentHbarEquiv", defaultValue = "1") int ratesCurrentHbarEquiv,
-        @ConfigProperty(value = "rates.currentCentEquiv", defaultValue = "12") int ratesCurrentCentEquiv,
-        @ConfigProperty(value = "rates.currentExpiry", defaultValue = "4102444800") long ratesCurrentExpiry,
-        @ConfigProperty(value = "rates.nextHbarEquiv", defaultValue = "1") int ratesNextHbarEquiv,
-        @ConfigProperty(value = "rates.nextCentEquiv", defaultValue = "15") int ratesNextCentEquiv,
-        @ConfigProperty(value = "rates.nextExpiry", defaultValue = "4102444800") long ratesNextExpiry,
-        @ConfigProperty(value = "system.entityExpiry", defaultValue = "1812637686") long systemEntityExpiry,
-        @ConfigProperty(value = "throttleDefsJson.resource", defaultValue = "throttles.json")
+        @ConfigProperty(value = "rates.currentHbarEquiv", defaultValue = "1") @NetworkProperty
+                int ratesCurrentHbarEquiv,
+        @ConfigProperty(value = "rates.currentCentEquiv", defaultValue = "12") @NetworkProperty
+                int ratesCurrentCentEquiv,
+        @ConfigProperty(value = "rates.currentExpiry", defaultValue = "4102444800") @NetworkProperty
+                long ratesCurrentExpiry,
+        @ConfigProperty(value = "rates.nextHbarEquiv", defaultValue = "1") @NetworkProperty int ratesNextHbarEquiv,
+        @ConfigProperty(value = "rates.nextCentEquiv", defaultValue = "15") @NetworkProperty int ratesNextCentEquiv,
+        @ConfigProperty(value = "rates.nextExpiry", defaultValue = "4102444800") @NetworkProperty long ratesNextExpiry,
+        @ConfigProperty(value = "system.entityExpiry", defaultValue = "1812637686") @NetworkProperty
+                long systemEntityExpiry,
+        @ConfigProperty(value = "throttleDefsJson.resource", defaultValue = "throttles.json") @NetworkProperty
```

Comment:
This looks like a file name, not sure if that is really a network value or a node-specific value.

Reply:
Updated.